common/commands/config.go

```diff
@@ -330,7 +330,7 @@ func getSshKeyPath(details *config.ServerDetails) error {
 
 	// If path still not provided, return and warn about relying on agent.
 	if details.SshKeyPath == "" {
-		log.Info("SSH Key path not provided. You can also specify a key path using the --ssh-key-path command option. If no key will be specified, you will rely on ssh-agent only.")
+		log.Info("SSH Key path not provided. You will rely on ssh-agent only.")
```

Comment:
...You will rely on ssh-agent only. --> ...The ssh-agent (if active) will be used.